packages/material-ui-lab/src/Masonry/Masonry.d.ts

```diff
@@ -0,0 +1,50 @@
+import { ResponsiveStyleValue, SxProps } from '@material-ui/system';
+import { OverridableComponent, OverrideProps } from '@material-ui/core/OverridableComponent';
+import { Theme } from '@material-ui/core/styles';
+import { MasonryClasses } from './masonryClasses';
+
+export interface MasonryTypeMap<P = {}, D extends React.ElementType = 'div'> {
+  props: P & {
+    /**
+     * The content of the component, normally `<MasonryItem />`s.
+     */
+    children: NonNullable<React.ReactNode>;
+    /**
+     * Override or extend the styles applied to the component.
+     */
+    classes?: Partial<MasonryClasses>;
+    /**
+     * Number of columns.
+     * @default 4
+     */
+    cols?: ResponsiveStyleValue<number | string>;
```

Comment:
For consistency? https://deploy-preview-27169--material-ui.netlify.app/components/grid/#columns
```suggestion
    columns?: ResponsiveStyleValue<number | string>;
```